lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -75,6 +75,9 @@ def on_resbody(node)
           preferred_name = preferred_name(offending_name)
           return if preferred_name.to_sym == offending_name
 
+          # check variable shadowing for exception variable
+          return if check_shadowed_variable_name(node)
```

Comment:
Can you update the caller name as well?

```suggestion
          return if shadowed_variable_name?(node)
```